helm/ph-ee-kenya/templates/config.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ph-ee-config
+data:
```

Comment:
@fynmanoj Why can’t we add these files to the ph-ee-engine chart itself? With every new chart with the parent chart as ph-ee-engine we have to add these *.properties files if these files are added in ph-ee-engine then we don't have to duplicate this part.

Please share some insights.

Reply:
these property files has to be use-case specific. so we could not abstract it into the base chart. It would be good if we can do it.